test/integration/domain.spec.ts

```diff
@@ -811,6 +811,13 @@ describe('Domain Layer', () => {
             accountsWithUnclaimedRewards
         }
     }`;
+
+    const proposalIsIndexed = async () => {
+      return (await sendQuery(getProposalRewards)).proposal.accountsWithUnclaimedRewards.length === 2;
+    };
```

Comment:
How about adding a test to check the actual proposal.accountsWithUnclaimedRewards ?

Reply:
for proposer.voters and stakers

Reply:
There is a test for that already...